### DISCUSSION TITLE
Overlay on GoPro12

### DISCUSSION THREAD

**Discussion by T39P** [gopro/labs#861] [Q&A]
Can't get the date and time overlay to work on GoPro12 for 8bit vids. Any help would be greatly appreciated. Does it work for anyone?

**Comment by dnewman-gpsw** [comment]
Only works in 4Kp30 16x9 8-bit.  I just tried it, and it is working with the default setup within https://gopro.github.io/labs/control/overlays/

**Reply by T39P** [reply]
Somehow works now and even in lower res with horizon stabilization, thanks.

**Reply by dnewman-gpsw** [reply]
It will also work at 1080p50/60/30/25/24
